docs/documentation/features.md

```diff
@@ -223,6 +222,27 @@ public class DeploymentReconciler
   }
 ```
 
+## Max Interval Between Reconciliations
+
+In case informers are all in place and reconciler is implemented correctly the reconciliation is triggered when needed,
```

Comment:
Maybe something like:

```
The reconciliation is supposed to be self-sufficient to cover all the use-cases, however ...
```

Reply:
will improve the wording, but this does not mean the same, also little bit abstract imho.

Reply:
The point is that event sources are needed to have a correct implementation.